evap/staff/templates/staff_semester_view_evaluation.html

```diff
@@ -188,6 +188,56 @@
             <a href="{% url 'staff:evaluation_copy' evaluation.id %}" class="btn btn-sm btn-light" data-bs-toggle="tooltip" title="{% translate "Copy" %}">
                 <span class="fas fa-copy"></span>
             </a>
+            {% if not evaluation.has_exam %}
+                <confirmation-modal type="submit" id="exam_creation_modal" name="evaluation_id" value="{{ evaluation.id }}" confirm-button-class="btn-primary" form="exam_creation_form_{{ evaluation.id }}">
+                    <span slot="title">{% translate 'Create exam evaluation' %}</span>
+                    <span slot="action-text" id="submit_button">{% translate 'Create exam evaluation' %}</span>
+                    <span slot="question">
+                        {% blocktranslate trimmed %}
+                            Create an exam evaluation based on this evaluation. This will copy all the participants and contributors from the original evaluation. It will set the weight of the original evaluation to 9 and its end date will be set to the day before the exam.
+                        {% endblocktranslate %}
+                    </span>
+                    <div slot="extra-inputs">
+                        <label for="exam_date">{% translate 'Exam Date:' %}</label>
+                        <input type="date" id="exam_date" name="exam_date" data-must-before="{{ evaluation.vote_start_datetime }}" class="form-control" required form="exam_creation_form_{{ evaluation.id }}"/>
```

Comment:
The `id="exam_date"` attribute is problematic, because this id will occur multiple times in the HTML, which is forbidden. After the change to the event handler from my last comment, the id is only needed for linking the `input` and `label` elements together - this can also be achieved by moving the input element into the label element, like `<label>Exam Date: <input ...></label>` (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label).

Alternatively, we could mix the evaluation id into it (`exam_date_{{ evaluation.id }}`), but I would prefer the first approach